packages/moxb/src/table/TableColumnImpl.ts

```diff
@@ -17,17 +17,26 @@ export interface TableColumnOptions extends BindOptions {
 export class TableColumnImpl extends BindImpl<TableColumnOptions> implements TableColumn {
     readonly column: string;
     readonly tableColumn: string;
+
     constructor(impl: TableColumnOptions, private readonly table: Table<any>) {
         super({ ...impl, id: table.id + '.' + impl.id });
+        makeObservable(this);
         this.column = impl.column || impl.id;
         this.tableColumn = impl.tableColumn || this.column;
     }
 
+    // We need to separate the actual method triggering the search from the public API,
+    // so that we can use a jest spy on the public API.
+    // (The mobx-wrapped action can't be wrapped by the spy.)
     @action.bound
```

Comment:
I think the bind should be on the toggleSort

Reply:
The problem is that we can't use the jest spy to wrap a method that is actually decorated as an action.
That's why I added another layer of wrapper. This way the actual method (`private _toggleSort`()) is decorated as an action, and the outer wrapper (`toggleSort()`) is used for testing by the spy.

Reply:
We can come up with another solution; the constraint is that we must be able to put a spy to above or below the actual action.